.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
```

Comment:
This is the only file that's run at test execution time. `Dockerfile` and all the other files are only when building the `freelawproject/courtlistener-testing` Docker image. This is done once and only need to be rebuild when there are actual dependency changes.